docs/docs/Tutorials/ROS2/PythonLinearDamperExample.md

```diff
@@ -240,13 +240,21 @@ names along with other changes according to the previous tutorial.
 
 ## Try It Out
 
+Make sure to build and source your workspace. This tutorial assumes you cloned your package to
+`~/controller_ws/src` and you have sourced `mbari_wec_gz` and `mbari_wec_utils`
+```
+$ cd ~/controller_ws
+$ colcon build
+$ source install/local_setup.bash
+```
+
 We will be using `ros2 launch` and `launch/controller.launch.py` to run our new controller.
 
-To run the controller along with the simulation, first source your workspace. Then, launch your
+To run the controller along with the simulation, launch your
 controller:  
-`$ ros2 launch mbari_wec_linear_damper_py controller.launch.py`
+`$ ros2 launch mbari_wec_linear_damper_cpp controller.launch.py`
```

Comment:
Umm this should be `mbari_wec_linear_damper_py`

Reply:
I'm not sure what you mean here.

Reply:
Maybe she thought it's a typo, based on the file extension? 😅 It's this file https://github.com/mbari-org/mbari_wec_template_cpp/blob/main/launch/controller.launch.py